tidb-server/main.go

```diff
@@ -35,6 +38,9 @@ var (
 )
 
 func main() {
+	tidb_metrics.RegMetircs(metrics.NewRegistry())
```

Comment:
You should put these two lines into a separated goroutine otherwise it will block main thread.